appserver/concurrent/concurrent-impl/src/main/java/org/glassfish/concurrent/runtime/ContextSetupProviderImpl.java

```diff
@@ -387,6 +387,24 @@ private void readObject(java.io.ObjectInputStream in) throws IOException, ClassN
         if (!nullTransactionManager) {
             transactionManager = concurrentRuntime.getTransactionManager();
         }
+        try {
+            this.requestTracing = Globals.getDefaultHabitat().getService(RequestTracingService.class);
+        } catch (NullPointerException ex) {
+            logger.log(Level.INFO, "Error retrieving Request Tracing service "
+                    + "during initialisation of Concurrent Context - NullPointerException");
```

Comment:
Nothing is less helpful than log entry about exception without a stacktrace. And misleading message :)

I propose "Error retrieving _<service>_ for Concurrent Context", ex.